src/components/tickets/ticket.service.ts

```diff
@@ -41,6 +42,21 @@ export const createTicket = asyncWrapper(
     next()
   })
 
+export const sendEmailToTicketAdmins = asyncWrapper(
+  async (req: Request, res: Response, next: NextFunction) =>  {
+    const ticket = req.body
+
+    const emailRecepients = await User.query().where('role', '=', 'TICKET_ADMIN')
```

Comment:
I believe we have an enum defined for user roles, better to use that here. Also, if we can use the object syntax in the where call, it would be preferred (like `.where{ role: UserRoles.TICKET_ADMIN }` or smth. like that).